============================= test session starts ==============================
platform linux -- Python 3.10.12, pytest-8.3.3, pluggy-1.5.0
rootdir: /home/wilkinson/Documents/TCC/TG
collected 5 items

QuixBugs/python_testcases/test_wrap.py FFFFF                             [100%]

=================================== FAILURES ===================================
_______________________ test_wrap[input_data0-expected0] _______________________

input_data = ['The leaves did not stir on the trees, grasshoppers chirruped, and the monotonous hollow sound of the sea rising up f...thing except what we think or do ourselves when we forget our human dignity and the higher aims of our existence.', 50]
expected = ['The leaves did not stir on the trees, grasshoppers', ' chirruped, and the monotonous hollow sound of the', ' sea ris...So it must have', ' sounded when there was no Yalta, no Oreanda here;', ' so it sounds now, and it will sound as', ...]

    @pytest.mark.parametrize("input_data,expected", testdata)
    def test_wrap(input_data, expected):
>       assert wrap(*input_data) == expected
E       AssertionError: assert ['The leaves ...l sound', ...] == ['The leaves ...ound as', ...]
E         
E         At index 0 diff: 'The leaves did not stir on the trees,' != 'The leaves did not stir on the trees, grasshoppers'
E         Left contains one more item: 'existence.'
E         Use -v to get more diff

QuixBugs/python_testcases/test_wrap.py:15: AssertionError
_______________________ test_wrap[input_data1-expected1] _______________________

input_data = ['The leaves did not stir on the trees, grasshoppers chirruped, and the monotonous hollow sound of the sea rising up f...thing except what we think or do ourselves when we forget our human dignity and the higher aims of our existence.', 20]
expected = ['The leaves did not', ' stir on the trees,', ' grasshoppers', ' chirruped, and the', ' monotonous hollow', ' sound of the sea', ...]

    @pytest.mark.parametrize("input_data,expected", testdata)
    def test_wrap(input_data, expected):
>       assert wrap(*input_data) == expected
E       AssertionError: assert ['The leaves ...the sea', ...] == ['The leaves ...the sea', ...]
E         
E         At index 1 diff: 'stir on the trees,' != ' stir on the trees,'
E         Use -v to get more diff

QuixBugs/python_testcases/test_wrap.py:15: AssertionError
_______________________ test_wrap[input_data2-expected2] _______________________

input_data = ['The leaves did not stir on the trees, grasshoppers chirruped, and the monotonous hollow sound of the sea rising up f...thing except what we think or do ourselves when we forget our human dignity and the higher aims of our existence.', 80]
expected = ['The leaves did not stir on the trees, grasshoppers chirruped, and the monotonous', ' hollow sound of the sea rising ...his constancy, in this complete', ' indifference to the life and death of each of us, there lies hid, perhaps, a', ...]

    @pytest.mark.parametrize("input_data,expected", testdata)
    def test_wrap(input_data, expected):
>       assert wrap(*input_data) == expected
E       AssertionError: assert ['The leaves ...haps, a', ...] == ['The leaves ...haps, a', ...]
E         
E         At index 0 diff: 'The leaves did not stir on the trees, grasshoppers chirruped, and the' != 'The leaves did not stir on the trees, grasshoppers chirruped, and the monotonous'
E         Left contains one more item: 'existence.'
E         Use -v to get more diff

QuixBugs/python_testcases/test_wrap.py:15: AssertionError
_______________________ test_wrap[input_data3-expected3] _______________________

input_data = ['The leaves did not stir on the trees, grasshoppers chirruped, and the monotonous hollow sound of the sea rising up f...thing except what we think or do ourselves when we forget our human dignity and the higher aims of our existence.', 77]
expected = ['The leaves did not stir on the trees, grasshoppers chirruped, and the', ' monotonous hollow sound of the sea rising ...e. And in this constancy, in this', ' complete indifference to the life and death of each of us, there lies hid,', ...]

    @pytest.mark.parametrize("input_data,expected", testdata)
    def test_wrap(input_data, expected):
>       assert wrap(*input_data) == expected
E       AssertionError: assert ['The leaves ...es hid,', ...] == ['The leaves ...es hid,', ...]
E         
E         At index 1 diff: 'monotonous hollow sound of the sea rising up from below, spoke of the peace,' != ' monotonous hollow sound of the sea rising up from below, spoke of the peace,'
E         Left contains one more item: 'existence.'
E         Use -v to get more diff

QuixBugs/python_testcases/test_wrap.py:15: AssertionError
_______________________ test_wrap[input_data4-expected4] _______________________

input_data = ['The leaves did not stir on the trees, grasshoppers chirruped, and the monotonous hollow sound of the sea rising up f...hing except what we think or do ourselves when we forget our human dignity and the higher aims of our existence.', 140]
expected = ['The leaves did not stir on the trees, grasshoppers chirruped, and the monotonous hollow sound of the sea rising up f...a, mountains, clouds, the open sky - Gurov thought how in reality everything is beautiful in this world when one', ...]

    @pytest.mark.parametrize("input_data,expected", testdata)
    def test_wrap(input_data, expected):
>       assert wrap(*input_data) == expected
E       AssertionError: assert ['The leaves ...hen one', ...] == ['The leaves ...hen one', ...]
E         
E         At index 1 diff: 'peace, of the eternal sleep awaiting us. So it must have sounded when there was no Yalta, no Oreanda here; so it sounds now, and it will' != ' peace, of the eternal sleep awaiting us. So it must have sounded when there was no Yalta, no Oreanda here; so it sounds now, and it will'
E         Left contains one more item: 'existence.'
E         Use -v to get more diff

QuixBugs/python_testcases/test_wrap.py:15: AssertionError
=========================== short test summary info ============================
FAILED QuixBugs/python_testcases/test_wrap.py::test_wrap[input_data0-expected0]
FAILED QuixBugs/python_testcases/test_wrap.py::test_wrap[input_data1-expected1]
FAILED QuixBugs/python_testcases/test_wrap.py::test_wrap[input_data2-expected2]
FAILED QuixBugs/python_testcases/test_wrap.py::test_wrap[input_data3-expected3]
FAILED QuixBugs/python_testcases/test_wrap.py::test_wrap[input_data4-expected4]
============================== 5 failed in 0.04s ===============================
Different AST
